### DISCUSSION TITLE
Crazyradio PA standalone use

### DISCUSSION THREAD

**Discussion by mzhouURI** [bitcraze/discussions#421] [Q&A]
Hi, 

We are looking for a easy to use usb dongle for long range communication.
I am wondering if we can configure the Crazyradio PA as a regular usb device so we can get system status from another Crazyradio PA on a robot?
If this is doable, is there any specific setting we need to do?

Thanks. 
Mingxi

**Comment by ataffanel** [comment]
It is open-source so everything is possible, but maybe not out of the box :-).

For some context: nRF24 radios implements two modes, PTX for the main sender/ack receiver and PRX for the main receiver/ack sender. With Crazyradio and Crazyflie the radios are configured in PTX and PRX mode respectively.

The latest version of the [Crazyradio firmware](https://github.com/bitcraze/crazyradio-firmware) (needs to be compiled and flashed from git), implements PRX mode which would allow to receive packets. This way the radio could, in theory, be used between two computers/robot. Though I have never tested it and there will be a need of some extra python code to handle all the details of the communication (for example to handle packet lost if needed). The Crazyradio python lib is the easiest way to use the Crazyradio. There is no other USB mode implemented so far (if you feel adventurous the firmware can be changed to accommodate any protocols, but this is much more work...).

As a side questions: what do you mean by "a regular usb device". I am currently looking at making the successor of the Crazyradio PA and it would be interesting to understand what type of interface would be more natural when communicating between robots.
